prime-router/src/main/resources/db/migration/R__report_lineage_report_id_indexes.sql

```diff
@@ -9,11 +9,11 @@
  */
 
 CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_report_lineage_parent_report_id
-  ON report_lineage
-  USING btree(parent_report_id)
+    ON report_lineage
+        USING btree(parent_report_id)
 ;
 
 CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_report_lineage_child_report_id
-  ON report_lineage
-  USING btree(child_report_id)
+    ON report_lineage
+        USING btree(child_report_id)
 ;
```

Comment:
If you use `R__` instead of `Vxx__` as the prefix to your flyway script it will always rerun your script for you on each migration, which is handy to know in the future. It of course means your script *MUST* be idempotent. This was another suggestion to try. So was adding `NT` to the front, which also __DID NOT__ work.